migrate/live-migration/live-migration-faq.md

```diff
@@ -19,6 +19,41 @@ the migration. The logs of these processes can be helpful for troubleshooting
 unexpected behavior. You can find these logs in the `<volume_mount>/logs` directory.
 
 
+## Source database does not have the latest TimescaleDB version
+
+Live migration requires the TimescaleDB version in the source database to be the same as
+that in the target database (Timescale Cloud). However, in certain instances, the TimescaleDB
+version installed in the target database may differ from that available in the source
+database, such as in the case of Managed Service for TimescaleDB. In such scenarios,
+instead of upgrading the source database, an alternative approach is to downgrade the
+TimescaleDB version in the target database to align it with the version available in
+the source database.
+
+
+This can be achieved by first uninstalling the TimescaleDB extension from the
+target database using the following command:
+
+```sql
+DROP EXTENSION timescaledb;
+```
+
+Then, reinstall TimescaleDB extension with the desired version:
+
+```sql
+CREATE EXTENSION timescaledb VERSION '<version>';
+```
+
+<Highlight type="note">
+You might need to reconnect `psql` with the `-X` flag to prevent reading the startup file.
+</Highlight>
+
+To determine which TimescaleDB versions are available on Timescale Cloud and can
+be used for the downgrade, execute the following query:
+
+```sql
+SELECT version FROM pg_available_extension_versions WHERE name = 'timescaledb' ORDER BY 1 DESC;
+```
```

Comment:
```suggestion

## Source database is not running the latest version of TimescaleDB

When you migrate a [self-hosted][self hosted] or [Managed Service for TimescaleDB (MST)][mst] 
database to Timescale, the source database and the destination 
[Timescale Service][timescale-service] must run the same version of TimescaleDB. 

Before you start [Live migration][live migration]:

<Procedure >

1. Check the version of TimescaleDB running on the source database and the 
   target Timescale Service:

    ```sql
    \dx timescaledb
    ```
   
    You see the following:
     ```sql
        Name     | Version |   Schema   |                             Description
    -------------+---------+------------+---------------------------------------------------------------------
     timescaledb | x.y.z   | public     | Enables scalable inserts and complex queries for time-series data
    (1 row)
    ```
 
1. If the version of TimescaleDB on the source database is lower than your Timescale Service, either:

   - **Downgrade**: reinstall an older version of TimescaleDB on your Timescale
      Service that matches the source database:
   
     1. Connect to your Timescale Service and check the versions of TimescaleDB available:
        
        ```sql
        SELECT version FROM pg_available_extension_versions WHERE name = 'timescaledb' ORDER BY 1 DESC;
        ```
          
     2. If an available TimescaleDB release matches your source database: 
        
        1. uninstall TimescaleDB from your Timescale Service:
    
           ```sql
           DROP EXTENSION timescaledb;
           ```
    
        1. Reinstall the correct version of TimescaleDB:
    
           ```sql
           CREATE EXTENSION timescaledb VERSION '<version>';
           ```
    
        <Highlight type="note">
        
        To prevent reading the startup file, reconnect to your Timescale Service using `psql -X` .
     
        </Highlight>

    
   - **Upgrade**: for self-hosted databases, 
     [upgrade TimescaleDB][self hosted upgrade] to match your Timescale Service. 



</ Procedure>


[live migration]: /migrate/:currentVersion:/live-migration/
[self hosted]: /self-hosted/:currentVersion:/
[self hosted upgrade]: /self-hosted/:currentVersion:/upgrades/
[mst]: /managed/:currentVersion:/
[timescale-service]: https://console.cloud.timescale.com/dashboard/services


```

Reply:
@billy-the-fish thanks for the suggestions. I have implemented the changes.